flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/batch/table/CalcITCase.scala

```diff
@@ -217,6 +217,21 @@ class CalcITCase(
   }
 
   @Test
+  def testConjunctivePredicate(): Unit = {
```

Comment:
This is basically a pure code generation change, we really need no ITCase for that. Can you remove this one? We really should keep ITCases to a minimum. They add to much overhead.